uniflow-core/src/main/kotlin/io/uniflow/core/flow/data/ThrowableKt.kt

```diff
@@ -0,0 +1,40 @@
+package io.uniflow.core.flow.data
+
+fun Throwable.toThrowableKt(): ThrowableKt {
+    return ThrowableKt(message, cause)
+}
+
+fun throwableKt(t: Throwable) = t.toThrowableKt()
+
+open class ThrowableKt(val message: String? = null, val cause: ThrowableKt? = null) {
+
+    var error: Throwable? = null
+        private set
```

Comment:
nope, I don't want it in the data class as it's not comparable

Reply:
Really it doesn't matter: the class has an `error` property, regardless of it being declared in the constructor (which is just Kotlin's compactness) or in the class body.

Also, this is not a data class, because it's open.

Also, `var error` can and should be `val error`: otherwise it seems as if some private process of the class can set the `error`.